### PR TITLE
Added sign in and login views

### PR DESCRIPTION
**Pull request by aaandrew**
Added login/sign up views

Thing to note is I added a package that for extending css/js files (ie I use it to inject CSS into the head of the layout file). Caveat is extend blocks need to be added to each view file.

screen shot 2016-05-10 at 7 27 35 pm

### CODE REVIEW

**Comment by aaandrew** [comment]
screen shot 2016-05-11 at 8 45 07 pm

**Comment by eduardoramirez** [comment]
Thanks. looks a lot better.

File: config/express.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ const cookieParser = require('cookie-parser')
 const express = require('express')
 const expressValidator = require('express-validator')
 const handlebars = require('express-handlebars')
+const hbsExtend = require('express-handlebars-extend');
```
Comment (eduardoramirez) [inline config/express.js]: semi-colon

File: config/express.js [hunk 2]
```diff
@@ -16,7 +17,8 @@ module.exports = (app, db, config) => {
 
   app.set('title', config.app.name)
 
-  app.engine('html', handlebars({ defaultLayout: 'main', extname: '.html' }))
+  const hbs = hbsExtend(handlebars.create({defaultLayout: 'main', extname: '.html'}));
```
Comment (eduardoramirez) [inline config/express.js]: semi-colon

File: config/express.js [hunk 3]
```diff
@@ -16,7 +17,8 @@ module.exports = (app, db, config) => {
...
+  app.engine('html', hbs.engine);
```
Comment (eduardoramirez) [inline config/express.js]: semi-colon

File: config/express.js (hunk 2 above)
Comment (eduardoramirez) [inline config/express.js]: Also, can you move the variable up higher 

```javascript
  const root = app.get('root')
  const hbs = hbsExtend(handlebars.create({defaultLayout: 'main', extname: '.html'}))

  app.set('title', config.app.name)

  app.engine('html', hbs.engine)
  app.set('view engine', 'html')
```

**Comment by aaandrew** [comment]
updated

**Pull request merged**